### PR TITLE
About

### PR DESCRIPTION
**Pull request by codyloyd**
did some cleaning up.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -23,12 +23,8 @@
           <%= render "layouts/#{flash[:partial]}" %>
         <% end %>
 
-        <div class="container">
-          <div class="row">
-            <%= render partial: 'shared/flash_helper', flash: flash %>
-            <%= yield %>
-          </div><!--/row-->
-        </div> <!-- /container -->
+        <%= render partial: 'shared/flash_helper', flash: flash %>
```
Comment (arun1595) [inline app/views/layouts/application.html.erb]: Nit. Use

```
<%= render 'shared/flash_helper', flash: flash %>
```

**Comment by KevinMulhern** [comment]
👍 LGTM

File: app/assets/stylesheets/components/about.scss [hunk 2]
```diff
@@ -134,13 +134,13 @@
   .contribute {
     color: #999;
     background: linear-gradient(white, $light-grey);
-
+    margin: 0;
     .cta {
         text-decoration: none;
         background-color: $button-primary;
         color: #ffffff;
```
Comment (arun1595) [inline app/assets/stylesheets/components/about.scss]: Nit.

You didn't make this change but can you please replace the hardcoded color with the `$white` variable defined in `settings.scss`?

File: app/assets/stylesheets/components/about.scss [hunk 3]
```diff
@@ -134,13 +134,13 @@
   .contribute {
     color: #999;
```
Comment (arun1595) [inline app/assets/stylesheets/components/about.scss]: Can you define a variable for this color in `settings.scss` and then use it here?

**Comment by arun1595** [comment]
@codyloyd In the beliefs section, there's too much space between the icon and content. Please have a look at the mockup.

File: app/views/static_pages/about/_contribute.html.erb [hunk 4]
```diff
@@ -1,9 +1,11 @@
-<div class="row contribute">
-  <div class="col">
-    <h2 class="text-center">
-      Help the Odin Project stay current and meaningful to all future students, please contribute
-    </h2>
+<div class="contribute">
+  <div class="container">
+    <div class="row">
+      <h2 class="text-center">
```
Comment (arun1595) [inline app/views/static_pages/about/_contribute.html.erb]: This heading should have the following styles:

```css
font-weight: 300;
font-size: 36px;
letter-spacing: 0.4px;
color: rgba(74, 74, 74, 0.7);
}
```

File: app/assets/stylesheets/components/about.scss (hunk 3 above)
Comment (codyloyd) [inline app/assets/stylesheets/components/about.scss]: looks like that's being overridden anyway... I just deleted it.

**Comment by codyloyd** [comment]
I made the fixes you requested... have a lookie

File: app/views/static_pages/about/_intro.html.erb [hunk 5]
```diff
@@ -1,9 +1,11 @@
 <div class="row intro">
+<div class="container">
```
Comment (arun1595) [inline app/views/static_pages/about/_intro.html.erb]: Nit. Nest it one level down.

**Comment by KevinMulhern** [comment]
LGTM 👍

**Comment by codyloyd** [comment]
alright.. I think this one is ready to merge.  Arun do you have any more comments?

**Comment by arun1595** [comment]
@codyloyd The layout in the curriculum page got messed up a bit. I've pushed a commit to fix it

**Pull request merged**